test/e2e/memcached_test.go

```diff
@@ -116,6 +126,18 @@ func TestMemcached(t *testing.T) {
 			t.Fatal(err)
 		}
 	}
+	defer func() {
```

Comment:
I am not entirely sure if deleting rbac resource is necessary. Since deleting the namespace including everything inside it.